### PR TITLE
kill

### PR DESCRIPTION
**Pull request by sevnadj**
Sevag Nadjarian (sevnadj)

Added documentation for the "kill" command

### CODE REVIEW

File: _commands/basics/kill.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+kill
+--
+
+`kill` is used to send signal to a process or to kill a process. 
+We typically use kill -SIGNAL PID, where you know the PID of the process.
+ 
+
+~~~ bash
+$ kill -9 7243
```
Comment (mterwill) [inline _commands/basics/kill.md]: Do you really want the first example to be ``kill -9``? Those poor processes.

Comment (sevnadj) [inline _commands/basics/kill.md]: I thought -9 was the signal to end the process though. At least in 485 that's what we used

Comment (mterwill) [inline _commands/basics/kill.md]: If you just ``kill PID`` it sends SIGTERM, which gives the process a chance to clean up after itself.

Ref: http://www.gnu.org/software/libc/manual/html_node/Termination-Signals.html

Comment (sevnadj) [inline _commands/basics/kill.md]: Updated

File: _commands/basics/kill.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+username    7243  7222  9 22:43 pts/2    00:00:00 vim
+
+$ kill -9 7243
+~~~
+
+##### Break it down
+ * `ps -ef` will print out all the processes running on the user's machine.
+ Piping with `| grep vim` will only output processes with the name
+ given, in this case only vim processes are printed. `kill -9 7243` is using
+ SIGTERM -9 to send the end process (kill) signal to process 7243, which
```
Comment (mterwill) [inline _commands/basics/kill.md]: `kill -9` sends `SIGKILL` not `SIGTERM -9`. Emphasize you should only use `-9` if a process is non-responsive to a normal `kill`.

**Comment by the-alex** [comment]
Looks good & passes. Thank you so much!

**Comment by mterwill** [comment]
Boo @the-alex was still waiting on changes

**Pull request merged**